### DISCUSSION TITLE
How "reporter" label works in latency metrics?

### DISCUSSION THREAD

**Discussion by AleksanderBrzozowski** [istio/istio#47571] [Q&A] [answered]
I’ve wanted to ask how `reporter` label works, for example in `istio_request_duration_milliseconds_sum` metric. Here is a diagram that I prepared, and I want to know if I understand it correctly. Basically, if I only want to measure the latency between application Istio proxy and application, I could use `reporter=destination` . Am I right?

![image](https://github.com/istio/istio/assets/15987499/d346be0e-9689-423e-9143-81169f564ecf)

**Comment by howardjohn** [comment, marked as answer]
Yep, I think you have the correct understanding.
